op-e2e/e2eutils/geth/wait.go

```diff
@@ -16,10 +17,10 @@ import (
 	"github.com/ethereum/go-ethereum/rpc"
 )
 
-var (
-	// errTimeout represents a timeout
-	errTimeout = errors.New("timeout")
-)
+const errStrTxIdxingInProgress = "transaction indexing is in progress"
```

Comment:
nit: is it possible to import the error message direct from geth, so there's no chance of drift?

Reply:
It unfortunately is not (yet). As of now it's an [internal error](https://github.com/ethereum/go-ethereum/blob/7fd7c1f7dd9ba8d90399df2f080e4101ae37a255/internal/ethapi/errors.go#L68).